src/main/resources/lang/strings_en.properties

```
@@ -46,6 +46,8 @@ config.general.safety.wait_before_revive=Wait before revive (sec)
 config.general.safety.wait_before_revive.desc=Seconds to wait before reviving
 config.general.safety.wait_after_revive=Wait after revive (sec)
 config.general.safety.wait_after_revive.des=Seconds to wait after reviving, lets ship repair
+config.general.safety.instant_repair=Minimum amount of InstantRepairs to use
+config.general.safety.instant_repair.des=Minimum amount of InstantRepairs to use after revive - 0 to disable
```

```suggestion
config.general.safety.instant_repair=Use instant repairs when above
config.general.safety.instant_repair.desc=Use instant repairs to fully heal the ship after revive if there's at least these amount of instant repairs left
```
The comment of 0 to disable shouldn't be needed if it's turned into a checkbox+number style